packages/pn-personafisica-webapp/src/pages/NotificationDetail.page.tsx

```diff
@@ -183,8 +192,6 @@ const NotificationDetail = () => {
     }
   };
 
-  const isCancelled = notification.notificationStatus === NotificationStatus.CANCELLED;
-
   const hasDocumentsAvailable = isCancelled || !notification.documentsAvailable ? false : true;
```

Comment:
this condition what does? are you sure it is ok? check the code

Reply:
this condition verify if actual notification status is in cancelled